src/ProjectDataProvider.ts

```diff
@@ -183,6 +186,78 @@ export module ProjectDataProvider {
     });
   };
 
+  export const formPackagedependency = vscodeRootpath => {
+    let manifestRootFolderPath: string = vscodeRootpath;
+    //        manifestRootFolderPath = item.split('package.json')[0];
+    return new Promise((resolve, reject) => {
+      fs.readFile(
+        manifestRootFolderPath + 'package.json',
+        { encoding: 'utf-8' },
+        function(err, data) {
+          if (data) {
+            let packageDependencies = JSON.parse(data);
+            fs.readFile(
+              manifestRootFolderPath + 'target/npmlist.json',
+              { encoding: 'utf-8' },
+              function(err, data) {
+                if (data) {
+                  let packageListDependencies = JSON.parse(data);
+                  let packageDepKeys = [];
+                  if (
```

Comment:
can you convert this statement in one line?

Reply:
this is due to line length increases and prettier breaks it, IMO it's better than having big line